mlflow/environment_variables.py

```diff
@@ -210,3 +210,6 @@ def get(self):
 MLFLOW_ALLOW_FILE_URI_AS_MODEL_VERSION_SOURCE = _BooleanEnvironmentVariable(
     "MLFLOW_ALLOW_FILE_URI_AS_MODEL_VERSION_SOURCE", False
 )
+
+#: Private environment variable that should be set to True when running MLflow tests.
+_MLFLOW_OPENAI_TESTING = _BooleanEnvironmentVariable("MLFLOW_OPENAI_TESTING", False)
```

Comment:
```suggestion
```

This can be removed. `mlflow/environment_variables.py` already has this on master